imap_processing/cdf/config/imap_glows_l1a_variable_attrs.yaml

```diff
@@ -1,358 +1,523 @@
 
 imap_sclk_next_pps:
   <<: *metadata_attrs
-  CATDESC: IMAP seconds for next PPS
-  FIELDNAM: IMAP seconds for next PPS
+  CATDESC: IMAP-clock seconds for next PPS
+  FIELDNAM: IMAP-clock seconds for next PPS
+  FILLVAL: *int_uint32
+  FORMAT: I11
+  LABLAXIS: IMAP seconds
+  UNITS: seconds
+  VALIDMAX: *int_uint32_min_one
 
 catbed_heater_active:
   <<: *metadata_attrs
-  CATDESC: Indicates if the heater is active
-  FIELDNAM: Heater active flag
+  CATDESC: Catbed-heater activity flag (1 - active, 0 - not active)
+  FIELDNAM: Catbed-heater activity flag
+  FILLVAL: -128
+  FORMAT: I2
+  LABLAXIS: Flag
 
 spin_period_valid:
   <<: *metadata_attrs
-  CATDESC: indicates if the spin phase is valid
-  FIELDNAM: valid spin phase flag
+  CATDESC: Spin-period-validity flag (1 - valid, 0 - invalid)
+  FIELDNAM: Spin-period-validity flag
+  FILLVAL: -128
+  FORMAT: I2
+  LABLAXIS: Flag
 
 spin_phase_at_next_pps_valid:
   <<: *metadata_attrs
-  CATDESC: Indicates if the spin phase at next PPS is valid
-  FIELDNAM: Spin phase at next PPS is valid
+  CATDESC: Spin-phase-at-next-PPS validity flag (1 - valid, 0 - invalid)
+  FIELDNAM: Spin-phase-at-next-PPS validity flag
+  FILLVAL: -128
+  FORMAT: I2
+  LABLAXIS: Flag
 
 spin_period_source:
   <<: *metadata_attrs
-  CATDESC: Flag - Spin period source
-  FIELDNAM: Flag - Spin period source
+  CATDESC: Spin-period-source flag (0 - from ITF, 1 - estimated by GLOWS AppSW)
+  FIELDNAM: Spin-period-source flag
+  FILLVAL: -128
+  FORMAT: I2
+  LABLAXIS: Flag
 
 spin_period:
   <<: *metadata_attrs
   CATDESC: Uint encoded spin period value
   FIELDNAM: Uint encoded spin period value
+  FILLVAL: *int_uint16
+  FORMAT: I6
+  LABLAXIS: Spin period
+  VALIDMAX: 50000 # TBC 15.38 s where 20.9712 s = 65535, rounded up
+  VALIDMIN: 45000 # TBC 14.63 s where 20.9712 s = 65535, rounded down
 
 spin_phase_at_next_pps:
   <<: *metadata_attrs
   CATDESC: Uint encoded next spin phase value
   FIELDNAM: Uint encoded next spin phase value
+  FILLVAL: *int_uint32
+  FORMAT: I6
+  LABLAXIS: Spin phase
+  VALIDMAX: *int_uint16
 
 number_of_completed_spins:
   <<: *metadata_attrs
-  CATDESC: Number of spins, from onboard
-  FIELDNAM: Number of spins, from onboard
+  CATDESC: Number of completed spins
+  FIELDNAM: Number of completed spins
+  FILLVAL: *int_uint32
+  FORMAT: I11
+  LABLAXIS: Num of spins
+  VALIDMAX: *int_uint32_min_one
 
 filter_temperature:
   <<: *metadata_attrs
-  CATDESC: Uint encoded temperature
-  FIELDNAM: Uint encoded temperature
+  CATDESC: Uint encoded filter temperature
+  FIELDNAM: Filter temperature
+  FILLVAL: *int_uint32
+  FORMAT: I6
+  LABLAXIS: Temperature
+  VALIDMAX: *int_uint16
+  DISPLAY_TYPE: time_series
 
 hv_voltage:
   <<: *metadata_attrs
-  CATDESC: Uint encoded voltage
-  FIELDNAM: Uint encoded voltage
+  CATDESC: Uint encoded CEM voltage
+  FIELDNAM: Uint encoded CEM voltage
+  FILLVAL: *int_uint32
+  FORMAT: I6
+  LABLAXIS: Voltage
+  VALIDMAX: 65635 # Is this a typo?
```

Comment:
I think this should should be the same as the rest of the UINT values. It might be worth pulling them out as a default grouping, but you already have them in each specific variable so it's up to you if that's worth doing